Buffer.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #ifndef BUFFER_SIZE
-#define BUFFER_SIZE 200
```

Comment:
This is still here

Reply:
So I would start a new branch every time I do a new PR? Or expect to do a new PR?